### DISCUSSION TITLE
How to use preset for climate devices.

### DISCUSSION THREAD

**Discussion by drpixel972** [Blackymas/NSPanel_HA_Blueprint#2314] [Q&A]
Hello,

I've configured a climate in the blueprint (main climate entity part).
However I can change the temperature and set the radiator to heat or off.
I would like to use the preset like in the card in lovelace.
I didn't find any information how to achieve that.
Is it possible ?

Best Regards,

**Comment by drpixel972** [comment]
Hello, 

Here is the climate page I get: 

![image](https://github.com/user-attachments/assets/81ad50e6-f40f-4710-9d89-1461317cc33d)


While others have this in the pages.

<img width="428" alt="image" src="https://github.com/user-attachments/assets/82412fbf-a251-436c-879e-c544c8a4467d">


In the dev tools, here are the states supported by the climate entity.
```

hvac_modes:
  - heat
  - "off"
min_temp: 7
max_temp: 22
target_temp_step: 0.5
preset_modes:
  - none
  - frost
  - eco
  - comfort
  - boost
  - activity
current_temperature: 21.7
```


I guess that I have only the two modes reported by the climate.
While other may have something like.

```
hvac_modes:
    "off"
    cool
    heat
 ...
```

Is there a way to have the presets : ['frost','eco','comfort','boost','activity', (or everything reported by the climate)] to be accesible like the hvac mode ?

Best Regards,

**Comment by lordcoxis** [comment]
I have the same issue, my heaters work with presets instead of HVAC modes... I think this would need development though, it's not currently supported.

**Comment by lordcoxis** [comment]
I opened an enhancement [issue](https://github.com/Blackymas/NSPanel_HA_Blueprint/issues/2394) for this.
